graphcode/generated_mpi/Makefile

```diff
@@ -10,24 +10,30 @@ LIBDIR = ../../../boost_1_85_0/lib
 
 
 # Define compiler and flags
-OMPI_CXX=g++ # Tells OpenMPI which compiler to use if there's more than one compiler (Comment out if not required)
-CXX = mpicxx
+ # Tells OpenMPI which compiler to use if there's more than one compiler (Comment out if not required)
+CXX = OMPI_CXX=g++ mpicxx
 CXXFLAGS = -g -std=c++17 -I$(INCDIR)
 LDFLAGS = -L$(LIBDIR)
 
 # Boost libraries
-LIBS = $(LIBDIR)/libboost_mpi.a $(LIBDIR)/libboost_serialization.a
-# LIBS = $(LIBDIR)/libboost_mpi-mt.a $(LIBDIR)/libboost_serialization-mt.a
+# LIBS = $(LIBDIR)/libboost_mpi.a $(LIBDIR)/libboost_serialization.a
```

Comment:
Why has the library changed to `*-mt.a`? As I recall, when we include the Boost library, it typically comes with `libboost_mpi.a`. Has there been a change in the library, or has the file included in the path been renamed locally?

Reply:
When I setup my MPI environment on Mac using `homebrew` the MPI library files installed was named with a `*-mt.a` suffix. It took me a while to figure this out. I feel this is specific to `homebrew` installations but I'm not very sure.